webapp-django/crashstats/symbols/management/commands/reencode-bad-sym-files.py

```diff
@@ -104,7 +126,17 @@ def handle(self, *args, **options):
                     assert key.content_type == 'text/plain', key.content_type
                     assert key.content_encoding == 'gzip', key.content_encoding
                     print filesizeformat(key.size).ljust(10),
-                    url = key.generate_url(expires_in=0, query_auth=False)
+
+                    # Instead of relying on key.generate_url(...)
+                    # which will produce a URL like
+                    # https://my.bucket.name.s3.amazonaws.com/key/name.ext
+                    # which can be troublesome for Python 2.7 on OSX to
+                    # GET because of the dots, we instead make our own
+                    # URL.
+                    url = 'https://s3.amazonaws.com/{}/{}'.format(
+                        key.bucket.name,
+                        key.name
+                    )
                     print url
```

Comment:
yep that was on purpose. On line 128 it prints the size and then after 10 spaces it prints the url.